### PR TITLE
test images: Adds OWNERS files for images (part 3)

### PR DESCRIPTION
**Pull request by claudiubelu**
**What type of PR is this?**

/kind documentation

/sig testing

**What this PR does / why we need it**:

Adds reviewers to the OWNERS files in the kubernetes/test/images folder. The reviewers are added automatically, based on their contributions on an image (>= 20% code churn).

Note that the code churn is taken into account for authors, and not committers.

Adds OWNERS files for: cuda-vector-add, nonewprivs, pets, redis, volume.

Script used to generate the OWNERS files: https://paste.ubuntu.com/p/smspRYH7fT/

Script logs: https://paste.ubuntu.com/p/CgT53yKBjf/

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by claudiubelu** [comment]
@mkumatag @jiayingz @jessfraz @bprashanth @rmmh @jsafrane Please give your ok that you are willing to be the approvers for these images.

/cc @fejta

File: test/images/volume/OWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+approvers:
+  - jsafrane
+  - mkumatag
```
Comment (jsafrane) [inline test/images/volume/OWNERS:3]: lgtm

**Pull request merged**